alarm/utils.py

```diff
@@ -33,3 +30,128 @@ def any_of_trade_pair_thresholds_is_broken(trade_pair):
             return True
 
     return False
+
+
+def refresh_thresholds_brakes_from_recent_candles_update(trade_pair):
+    thresholds = Threshold.objects.filter(trade_pair=trade_pair)
+    for threshold in Threshold.objects.filter(trade_pair=trade_pair):
+        if ThresholdBrake.objects.filter(threshold=threshold).exists():
+            continue
+    for threshold in thresholds:
+        try:
+            latest_threshold_brake = ThresholdBrake.objects.filter(threshold__trade_pair=trade_pair).latest(
+                'happened_at')
+            if latest_threshold_brake:
+                return False
+        except ThresholdBrake.DoesNotExist:
+            pass
+
+        ThresholdBrake.objects.create(threshold=threshold)
+
+
+def create_message_about_threshold_break():
+    all_threshold_breaks = ThresholdBrake.objects.all()
+    messages = {}
+    for threshold_break in all_threshold_breaks:
+        threshold = threshold_break.threshold
+
+        trade_pair = threshold.trade_pair
+
+        trade_pair_str = format_trade_pair_for_message(trade_pair)
+
+        threshold_prices = list(Threshold.objects.filter(trade_pair=trade_pair).values_list('price', flat=True))
```

Comment:
You got all threshold regardless of if they were broken or not here.

Reply:
++